go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/openshift/client-go v0.0.0-20221019143426-16aed247da5c
 	github.com/project-codeflare/appwrapper v0.27.0
 	github.com/project-codeflare/codeflare-common v0.0.0-20240930133152-11fd6e3be6b3
-	github.com/ray-project/kuberay/ray-operator v1.1.1
+	github.com/ray-project/kuberay/ray-operator v1.2.1
```

Comment:
Why did this change?

Reply:
This is brought by Kueue 0.8.3 dependency. That is why I had to rewrite it below.